include/tvm/runtime/data_type.h

```diff
@@ -107,6 +119,13 @@ class DataType {
   bool is_vector_bool() const { return is_vector() && bits() == 1; }
   /*! \return whether type is a Void type. */
   bool is_void() const { return code() == DataType::kHandle && bits() == 0 && lanes() == 0; }
+  bool is_scalable() const { return is_scalable_; }
+
+  DataType with_scalable_lanes() const {
+    int min_num_lanes = 128 / bits();
```

Comment:
macro? and does here only support vector size 128 or should be configurable between 128 - 2048?